config/config.go

```diff
@@ -176,9 +176,10 @@ func (u URL) MarshalYAML() (interface{}, error) {
 
 // Config is the top-level configuration for Prometheus's config files.
 type Config struct {
-	GlobalConfig  GlobalConfig    `yaml:"global"`
-	RuleFiles     []string        `yaml:"rule_files,omitempty"`
-	ScrapeConfigs []*ScrapeConfig `yaml:"scrape_configs,omitempty"`
+	GlobalConfig        GlobalConfig     `yaml:"global"`
+	AlertRelabelConfigs []*RelabelConfig `yaml:"alert_relabel_configs,omitempty"`
```

Comment:
Yeah, a new section would be best. I'd go more for `alerting`, as users might think that `alerts` is where you define alerts.

I'd also call this `alert_relabel_configs` to reduce confusion.